http-filter-example/http_filter_v2_integration_test.cc

```diff
@@ -0,0 +1,50 @@
+#include "test/integration/http_integration.h"
+#include "test/integration/utility.h"
+
+namespace Envoy {
+class HttpFilterSamplev2IntegrationTest : public HttpIntegrationTest,
+                                        public testing::TestWithParam<Network::Address::IpVersion> {
+public:
+  HttpFilterSamplev2IntegrationTest() : HttpIntegrationTest(Http::CodecClient::Type::HTTP1, GetParam()) {}
+  /**
+   * Initializer for an individual integration test.
+   */
+  void SetUp() override {
+    fake_upstreams_.emplace_back(new FakeUpstream(0, FakeHttpConnection::Type::HTTP1, version_));
+    registerPort("upstream_0", fake_upstreams_.back()->localAddress()->ip()->port());
+    createTestServer("http-filter-example/envoy-v2.yaml", {"http"});
```

Comment:
For v2 testing, we have a somewhat better approach, where configs are built progamatically. See https://github.com/envoyproxy/envoy/blob/master/test/integration/ratelimit_integration_test.cc#L14 for example.

Reply:
I can adopt this. Not a big deal. I have made changes. But build is failing because it is not able to refer the external dep
 external_deps = [
        "envoy_filter_network_http_connection_manager"
    ]
It expects some changes in WORKSPACE - can you point to me to the required change in WORKSPACE?

Reply:
ok. looks like it has been moved from http to network scope. May be I am working with old - should i update envoy in this and test?